### ISSUE TITLE
Bug disponibilité antérieure à la publication du jeu de données

### ISSUE THREAD

**Issue by NicolasBerthelot** [etalab/transport-site#2118]
Ce jeu de données a été publié le 15/02/2022 sur data.gouv.fr et référencé le même jour sur transport.data.gouv.fr
https://transport.data.gouv.fr/resources/78901#download-availability

Or, il apparaît que ses ressources sont détectées comme accessibles depuis un mois ce qui est impossible.

**Comment by fchabouis** [comment]
Oui, c'est une limitation du choix technique actuel : nous ne stockons en base que les périodes d'indisponibilités. Si rien n'est stocké, alors on dit que la ressource était dispo. Du coup lors de l'arrivée d'une ressource dans le PAN, elle a une disponibilité sur 30 jours qui s'affiche tout en vert. Je serais d'avis de changer ça et de stocker activement les 2 infos : dispo et indispo. Ce qui permettra d'afficher correctement les ressources fraichement importées. cc @AntoineAugusti

**Comment by thbar** [comment]
On peut stocker les deux infos, ou bien avoir une astuce du type "je note dans un coin le premier moment où ça a commencé à être dispo + dernier test" si on tenait absolument à économiser la place (mais c'est plus bancal).

Perso je pense qu'il vaut mieux aller vers un modèle ou on note les deux, quitte à partir vers un système qui scale bien ça time TimescaleDB (extension Postgres) si ça ne tient plus à un moment.
